applications/MeshingApplication/custom_io/mmg_io.cpp

```diff
@@ -169,6 +169,19 @@ void MmgIO<TMMGLibrary>::WriteModelPart(ModelPart& rModelPart)
 /***********************************************************************************/
 /***********************************************************************************/
 
+template<MMGLibrary TMMGLibrary>
+std::string MmgIO<TMMGLibrary>::GetMmgVersion()
+{
+    KRATOS_TRY;
+
+    return mMmgUtilities.GetMmgVersion();
```

Comment:
Is it possible to keepthe backward compatible parts using this? @loumalouomega

Reply:
In one moment...

Reply:
Now should...